sample/app/src/index.tsx

```diff
@@ -4,10 +4,13 @@ import { StrictMode } from "react";
 import { createRoot } from "react-dom/client";
 import { App } from "./App.tsx";
 
-if (process.env.USE_MSW) {
-    import("../mocks/browser.ts").then(({ worker }) => {
-        worker.start();
-    });
+// This first condition is to avoid MSW to be bundled in production
+if (process.env.NODE_ENV === "development") {
```

Comment:
Couldn't it be avoided by not settings USE_MSW to true for production? Which wouldn't make sense anyway. The condition seems redundant.